src/readfish/plugins/_mappy.py

```diff
@@ -82,15 +82,12 @@ def validate(self) -> None:
         :return: None, if the Aligner is setup with valid paths and permissions
         """
         index: str = self.aligner_params["fn_idx_in"]
-        file_extensions = [".fasta", ".fna", ".fsa", ".fa", ".fastq", ".fq"]
-        file_extensions.extend([f"{f}.gz" for f in file_extensions])
-        file_extensions.append(".mmi")
+        file_extensions = ['.fasta', '.fna', '.fsa', '.fa', '.fastq',
+                           '.fq', '.fasta.gz', '.fna.gz', '.fsa.gz',
+                           '.fa.gz', '.fastq.gz', '.fq.gz', '.mmi']
```

Comment:
I would keep the removed lines here, makes updating this list much easier in the future.

Reply:
Done